bundles/binding/org.openhab.binding.harmonyhub/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="/org.openhab.io.harmonyhub/target/org.openhab.io.harmonyhub-1.7.0-SNAPSHOT.jar"/>
+	<classpathentry kind="lib" path="/org.openhab.io.harmonyhub/lib/harmony-java-client-1.0.3-all.jar"/>
```

Comment:
which dependencies are included in the -all.jar? Any chance to find a more fine-grained (in terms of dependencies) version of the client?